core/src/main/java/org/apache/accumulo/core/client/rfile/RFileScanner.java

```diff
@@ -406,7 +406,7 @@ public void close() {
       if (this.blockCacheManager != null) {
         this.blockCacheManager.stop();
       }
-    } catch (Exception e1) {
+    } catch (RuntimeException e1) {
```

Comment:
This catch block could probably be removed.

Reply:
`this.blockCacheManager.stop();` has the possibility to throw an `UnsupportedOperationException` so I can catch/throw that here to narrow it.